dist/src/main/dist/conf/hazelcast.xml

```diff
@@ -83,4 +82,11 @@
         <eviction max-size-policy="ENTRY_COUNT" size="10000" eviction-policy="LFU"/>
     </cache>
 
-</hazelcast>
\ No newline at end of file
+    <topic name="ReliableITopic*">
```

Comment:
I think it would be best to have a seperate section:

```
<topic>
    <reliable>
         <enabled>true</enabled>
         <ringbuffer>...</ringbuffer>
         <errorhandler>....<errorhandler>
    </reliable>
```